myuw/static/css/typography.less

```diff
@@ -33,7 +33,15 @@ hr {
     border-top: 1px solid #ddd;
 }
 
-/** Lists **/
+/*** Links ***/
+a { text-decoration: none; color: #4d66ca;
+    &:hover, &:focus { text-decoration: underline; color: #334baa; }
+    &:active { text-decoration: underline; color: @uw-metalic-gold; }
```

Comment:
All looks good, save for this gold color. Better to use a blue similar to #2e55f2 - a bright blue.